pom.xml

```diff
@@ -74,6 +74,26 @@
 			<version>1.18.30</version>
 			<scope>provided</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
why do we need this dependency? I was not expecting any dependency other than spring tests and junit/jupter

Reply:
Ok